app/Controllers/BaseController.php

```diff
@@ -54,5 +61,11 @@ public function initController(RequestInterface $request, ResponseInterface $res
         // Preload any models, libraries, etc, here.
 
         // E.g.: $this->session = \Config\Services::session();
+        $this->session = \Config\Services::session();
+    }
+
+    public function __construct()
+    {
+        $this->isLoggedIn = $this->session->get('userData');
```

Comment:
$isLogin() => check login status(true, false only).
$userData => $this->session->get('userData'); store the user data.

Reply:
done